ngraph_bridge/api.h

```diff
@@ -58,7 +58,8 @@ extern void StartLoggingPlacement();
 extern void StopLoggingPlacement();
 extern bool IsLoggingPlacement();
 
-extern std::set<string> GetDisabledOps();
+__attribute((abi_tag("cxx11"))) extern std::set<string> GetDisabledOps();
```

Comment:
Removing for now. It might make sense to apply the ABI tag to the entire namespace. There are other functions here which could potentially be used from across the ABI boundary.